images/terraform-google-gke-cluster/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Copyright 2018 The Jetstack contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM debian:buster-slim
+
+RUN apt-get -y update \
+&& apt-get -y install --no-install-recommends \
+   ca-certificates \
+   curl \
+   unzip \
+&& curl https://releases.hashicorp.com/terraform/0.11.13/terraform_0.11.13_linux_amd64.zip -o terraform.zip \
```

Comment:
(may be best in a follow up): you can make the terraform version number be configurable via the `build.yaml` file, and then we'll be able to build a 'variant' of this image for each terraform version to make upgrading in the future easier to track.